### PR TITLE
Feat/modules

### PR DESCRIPTION
**Pull request by akiyashkin-ronasit**
added iam, ecr, s3, eks-post-setup, k8s-addons and k8s-resources module according to task 1044 https://app.clickup.com/t/24336023/PRD-1044

### CODE REVIEW

File: .devcontainer/README.md
Comment (eleonov) [inline .devcontainer/README.md:1]: please remove empty file

File: .vscode/README.md
Comment (eleonov) [inline .vscode/README.md:1]: please remove empty file

File: README.md [hunk 1]
```diff
@@ -31,4 +31,7 @@ We appreciate your contributions! We'll review your pull request and provide fee
 
 ## License
 
-This project is licensed under the [MIT License](LICENSE).
\ No newline at end of file
+This project is licensed under the [MIT License](LICENSE).
+<!-- BEGIN_TF_DOCS -->
```
Comment (eleonov) [inline README.md]: please remove useless block

File: modules/iam/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  sensitive   = false
+}
+
+output "developers_iam_role" {
+  value       = module.developers_iam_role
+  description = "Developer role details to use in other modules"
+  sensitive   = false
+}
+
+output "ECR_pull_policy" {
```
Comment (eleonov) [inline modules/iam/outputs.tf]: this is not the latest version of the module

**Comment by eleonov** [comment]
@akiyashkin-ronasit please, generate docs for each module separatly

```sh
# go to the module
cd modules/ecr/

# run docs gen command
terraform-docs . -c ../../.terraform.docs.yml
```

File: .devcontainer/README.md
Comment (akiyashkin-ronasit) [reply .devcontainer/README.md:1]: done

File: .vscode/README.md
Comment (akiyashkin-ronasit) [reply .vscode/README.md:1]: done

File: README.md (hunk 1 above)
Comment (akiyashkin-ronasit) [reply README.md]: useless block removed

File: modules/iam/outputs.tf (hunk 2 above)
Comment (akiyashkin-ronasit) [reply modules/iam/outputs.tf]: done

**Pull request merged**